feature-libs/checkout/scheduled-replenishment/components/checkout-place-order/checkout-place-order.component.ts

```diff
@@ -17,15 +17,15 @@ import {
 } from '@spartacus/core';
 import { LaunchDialogService, LAUNCH_CALLER } from '@spartacus/storefront';
 import { BehaviorSubject, merge, Subscription } from 'rxjs';
-import { CheckoutReplenishmentFormService } from '../../../scheduled-replenishment/components/services/checkout-replenishment-form-service';
+import { CheckoutReplenishmentFormService } from '../services/checkout-replenishment-form-service';
 
 @Component({
   selector: 'cx-place-order',
-  templateUrl: './place-order.component.html',
+  templateUrl: './checkout-place-order.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ScheduledReplenishmentPlaceOrderComponent
-  extends PlaceOrderComponent
+export class CheckoutScheduledReplenishmentPlaceOrderComponent
```

Comment:
Just want to make sure it's okay with you too, but the filename for like scheduled thank you and place order is fine to keep right? 

Don't need to prefix scheduled-replenishment and stuff right :d?

Reply:
I was debating with myself, and I decided to keep it short-ish. If it's located in the repl folder, it should be clear what the components is used for.
Also, not bad to have it like, if someone needs to update a template in the base, maybe they'll notice this cmp and update it as well.